proto/capten_plugins.proto

```diff
@@ -16,6 +16,7 @@ service capten_plugins {
     rpc GetTektonPipelines(GetTektonPipelinesRequest) returns (GetTektonPipelinesResponse) {}
     rpc CreateTektonPipelines(CreateTektonPipelinesRequest) returns (CreateTektonPipelinesResponse) {}
     rpc UpdateTektonPipelines(UpdateTektonPipelinesRequest) returns (UpdateTektonPipelinesResponse) {}
+    rpc DeleteTektonPipelines(DeleteTektonPipelinesRequest) returns(DeleteTektonPipelinesResponse) {}
     rpc SyncTektonPipelines(SyncTektonPipelinesRequest) returns (SyncTektonPipelinesResponse) {}
```

Comment:
GetTektonProjects --> GetTektonProject
CreateTektonPipelines -> CreateTektonPipeline, same for UPDATE and DELETE APIS
Implement UnRegisterTektonProject

Remove SyncTektonPipelines, move implementation to RegisterTektonProject